lib/models/person_model.dart

```diff
@@ -0,0 +1,284 @@
+import 'dart:convert';
+
+class Person {
```

Comment:
Do we need all these attributes? / is there a way to get only the attributes we need (and maybe add other attributes if we discover we need more) ?
cc @ilie-lupu-ts @szaszgyozo